cli/turbo.json

```diff
@@ -14,9 +14,11 @@
       ],
 
       "inputs": [
-        "**/*.go",
-        "**/*_test.go",
-        "../crates/turborepo*/**" // Rust crates
+        "{internal,cmd}/**/*.go",
+        "!**/*_test.go",
```

Comment:
we don't want to build again when unit tests change?

Reply:
I don't think so...they don't influence the build. If they uncovered a bug, there would have to be a corresponding source code change for a fix, which would trigger a build.